src/components/Calculator.jsx

```diff
@@ -33,25 +34,29 @@ const Calculator = () => {
 
   return (
 
-    <div className="calculator">
-      <DisplayInp />
-      <Operator className="operationContainer" handleClick={setValue} />
-      <div className="numbers">
-        <button type="button" onClick={() => setValue('7')}>7</button>
-        <button type="button" onClick={() => setValue('8')}>8</button>
-        <button type="button" onClick={() => setValue('9')}>9</button>
-        <button type="button" onClick={() => setValue('4')}>4</button>
-        <button type="button" onClick={() => setValue('5')}>5</button>
-        <button type="button" onClick={() => setValue('6')}>6</button>
-        <button type="button" onClick={() => setValue('1')}>1</button>
-        <button type="button" onClick={() => setValue('2')}>2</button>
-        <button type="button" onClick={() => setValue('3')}>3</button>
-      </div>
-      <div className="zeroDiv">
-        <button type="button" onClick={() => setValue('0')} className="zero">0</button>
-        <button type="button" onClick={() => setValue('.')}>.</button>
+    <>
+      <div className="calculator">
+        <DisplayInp />
+        <Operator className="operationContainer" handleClick={setValue} />
+        <div className="numbers">
+          <button type="button" onClick={() => setValue('7')}>7</button>
+          <button type="button" onClick={() => setValue('8')}>8</button>
+          <button type="button" onClick={() => setValue('9')}>9</button>
+          <button type="button" onClick={() => setValue('4')}>4</button>
+          <button type="button" onClick={() => setValue('5')}>5</button>
+          <button type="button" onClick={() => setValue('6')}>6</button>
+          <button type="button" onClick={() => setValue('1')}>1</button>
+          <button type="button" onClick={() => setValue('2')}>2</button>
+          <button type="button" onClick={() => setValue('3')}>3</button>
+        </div>
+        <div className="zeroDiv">
+          <button type="button" onClick={() => setValue('0')} className="zero">0</button>
+          <button type="button" onClick={() => setValue('.')}>.</button>
+        </div>
       </div>
-    </div>
+
+      <DisplayApi />
+    </>
```

Comment:
- [ ] Kindly move your `DisplayApi` to the `App.js` component for better structure. `Calculator` is a separate component, `DisplayApi` is also a separate component, and  `DisplayApi` does not have anything to do inside the `Calculator` component. Kindly make it to look something similar to this. :point_down: 

```
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}

export default App

```